spec/spec_helper.rb

```diff
@@ -13,4 +13,27 @@
   end
 
   config.shared_context_metadata_behavior = :apply_to_host_groups
+
+RSpec.configure do |config|
```

Comment:
pls move this configuration to a separate file, e.g.: `spec/support/config/carrier_wave.rb`